test/threadsafe_function/threadsafe_function_existing_tsfn.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const assert = require('assert');
+
+const buildType = process.config.target_defaults.default_configuration;
+
+module.exports = Promise.all([
+  test(require(`../build/${buildType}/binding.node`)),
+  test(require(`../build/${buildType}/binding_noexcept.node`))
+]);
+
+async function test(binding) {
+  const testCall = binding.threadsafe_function_existing_tsfn.testCall;
+  let lastResult;
+  
+  const funct = function(arg0) {
+    // console.log("TSFN called with args", ...arguments);
```

Comment:
Could this line be removed?

Reply:
Addressed.

Reply:
Re-addressed due to test restructure. This function has been deleted.